config/forms/2021_2022.json

```diff
@@ -1228,11 +1226,7 @@
               }
             },
             "females_in_soft_age_range_in_pregnant_household_lead_age_value_check": {
-              "depends_on": [
-                {
-                  "female_in_pregnant_household_in_soft_validation_range?": true
-                }
-              ],
+              "depends_on": [{ "female_in_pregnant_household_in_soft_validation_range?": true }],
```

Comment:
Your editor seems to have changed the formatting/indentation. Or did you deliberately change it?

Reply:
Was deliberate, made it easier to use code folding to see where to make the changes